modules/subscriptions/subscriptions.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { GlobalCore } from "@/core/module/module.types";
+import subscriptiosService from "@/services/app/subscriptios.service";
+import styles from "./styles/subscriptions.module.css";
+
+const Subscriptions = () => {
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(false)
+
+  const handleSubscribe = async () => {
+    setIsLoading(true)
+    const { url } = await subscriptiosService.getSubscriptionLink();
+    if (url) {
+      window.location.href = url;
+    }
+    setIsLoading(false)
+  };
+
+  return (
+    <>
+      <main className={styles.mainContent}>
+        <h1 className={styles.pricingTitle}>
+          Find the pricing plan for your business
```

Comment:
Remember to use i18n :D  (just english is ok, will add rest of the languages afetr)